server/mailet/integration-testing/src/main/resources/smtpserver.xml

```diff
@@ -55,9 +55,9 @@
     </smtpserver>
     <smtpserver enabled="true">
         <jmxName>smtpserver-TLS</jmxName>
-        <bind>0.0.0.0:0</bind>
+        <bind>0.0.0.0:465</bind>
```

Comment:
This change makes some integration tests fail. 
`java.net.SocketException: Permission denied`

Reply:
reverted